tests/roles/ironic_adoption/tasks/main.yaml

```diff
@@ -0,0 +1,173 @@
+- name: deploy podified Ironic
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc patch openstackcontrolplane openstack --type=merge --patch '
+    spec:
+      ironic:
+        enabled: true
+        template:
+          images:
```

Comment:
Should remove once https://github.com/openstack-k8s-operators/tcib/pull/150 is available.